packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -734,9 +734,9 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
 
     if (!ev.defaultPrevented) {
       // When Edge + Narrator are used together (regardless of if the button is in a form or not), pressing
-      // "Enter" fires this method and not _onMenuKeyDown. Checking ev.nativeEvent.detail differentiates
-      // between a real click event and a keypress event.
-      const shouldFocusOnContainer = ev.nativeEvent.detail !== 0;
+      // "Enter" fires this method and not _onMenuKeyDown. Checking ev.nativeEvent.screenX and
+      // ev.nativeEvent.screenY differentiates between a real click event and a keypress event.
+      const shouldFocusOnContainer = ev.nativeEvent.screenX !== 0 && ev.nativeEvent.screenY !== 0;
```

Comment:
Are you sure that this will be 0 and not undefined or null?

Reply:
Yup, I doubled check, it's 0.